styleguide/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
this file can probably be removed, right?

Reply:
Done